### PR TITLE
Connect backend and frontend for Github Conditions Page

### PR DESCRIPTION
**Pull request by kuries**
Parent: #46  
Related to #22 

A couple of other commits related to this PR:

- [Fix owner.parent not found bug in fetch github conditions route.](https://github.com/MatrixDoctors/matrix-cerberus/pull/49/commits/b62c73cae8f13b817b05b65a62a7be157f22c3b7)
- [Use owner's access token to fetch repository permissions under a user.](https://github.com/MatrixDoctors/matrix-cerberus/pull/49/commits/5b7da28193db1fc125e0f76da74b4d700b71d361)
- [Add last_edited_by attribute in github org conditions to keep track of the last admin who modified the conditions.](https://github.com/MatrixDoctors/matrix-cerberus/pull/49/commits/4b795406b8d321e1ad4b46c247c362e401f07a5b)

### CODE REVIEW

File: frontend/src/Components/EditConditions.js [hunk 1]
```diff
@@ -56,13 +57,12 @@ export default function EditConditions({roomId, modalData, setModalData, roomCon
...
                 });
             });
-
-            setShowEditable(false);
+            toast.success("Succesfully updated!");
         })
-        .catch( (err) => {
-            console.error("Failed to save data", err);
-            handleClose();
+        .catch( () => {
```
Comment (Half-Shot) [inline frontend/src/Components/EditConditions.js:62]: Log the error still for debugging?

File: frontend/src/Pages/RoomGithubConditions/Components/EditConditions.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                "condition_type": conditionType,
+                "data": currentData
+            }
+
+            await axios.put(url, dataToBeSent)
+            .then(() => {
+                toast.success('Successfully updated!');
+            })
+            .catch((err) => {
+                toast.error("Failed to save data");
```
Comment (Half-Shot) [inline frontend/src/Pages/RoomGithubConditions/Components/EditConditions.js:46]: Log the error here?

**Review by Half-Shot — APPROVED** [review]
Looks good from a frontend perspective

File: backend/app/api/endpoints/rooms.py [hunk 3]
```diff
@@ -95,13 +100,70 @@ async def get_room_conditions(room_id: str):
     return JSONResponse(list_of_conditions)
 
 
+@router.post(
```
Comment (Cadair) [inline backend/app/api/endpoints/rooms.py:103]: Why is this a POST if it's only getting data out of the system?

**Review by Cadair — APPROVED** [review]

**Pull request merged**